bench/workgen/workgen.cxx

```diff
@@ -1106,13 +1105,40 @@ void Operation::kv_gen(bool iskey, uint32_t randomizer, uint64_t n,
     if (n > max)
         THROW((iskey ? "Key" : "Value") << " (" << n
           << ") too large for size (" << size << ")");
-    if (randomizer != 0) {
-        randomizer %= 1000;
-        snprintf(result, 6, ":%3.3d:", randomizer);
-        n -= RANDOMIZER_SIZE;
-        result += RANDOMIZER_SIZE;
-    }
```

Comment:
I think we these changes, we should get rid of the `RANDOMIZER_SIZE` definition.